minkyu/express_init.js

```diff
@@ -0,0 +1,41 @@
+const http = require('http');
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const dotenv = require('dotenv');
+dotenv.config();
+
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+    type: process.env. TYPEORM_CONNECTION,
+    host: process.env. TYPEORM_HOST,
+    port: process.env. TYPEORM_PORT,
+    username: process.env. TYPEORM_USERNAME,
+    password: process.env. TYPEORM_PASSWORD,
+    database: process.env. TYPEORM_DATABASE,
+})
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    });
+
+app = express();
```

Comment:
변수 (상수)를 선언하는 const가 빠져있는 것 같습니다.
```suggestion
const app = express();
```